gdx/src/com/badlogic/gdx/math/MathUtils.java

```diff
@@ -182,6 +182,24 @@ static public int randomSign () {
 		return 1 | (random.nextInt() >> 31);
 	}
 
+	/** Returns a random point inside the unit circle */
+	static public Vector2 randomUnitCirclePoint (Vector2 vector) {

```

Comment:
Wouldn't it make sense to change this method so that it actually returns only normalized vectors (length = 1). I think this method is mostly going to be used to create a random direction and it would make things easier if the vectors had all the same length. Otherwise at least rename the method so that it is clear that it does return points in the unit circle and not only those on the unit circle.